digitalocean/migratedata.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+set -e
+
+echo
+echo "Initializing data migration to the new block storage volume..."
+echo
+
+NEW_VOLUME_MOUNTING_POINT=/mnt/lamassu-data
+
+CERT_DIR=/etc/ssl/certs
+KEY_DIR=/etc/ssl/private
+CONFIG_DIR=/etc/lamassu
+
+LAMASSU_CA_PATH=$CERT_DIR/Lamassu_CA.pem
+CA_KEY_PATH=$KEY_DIR/Lamassu_OP_Root_CA.key
+CA_PATH=$CERT_DIR/Lamassu_OP_Root_CA.pem
+SERVER_KEY_PATH=$KEY_DIR/Lamassu_OP.key
+SERVER_CERT_PATH=$CERT_DIR/Lamassu_OP.pem
+MNEMONIC_DIR=$CONFIG_DIR/mnemonics 
+MNEMONIC_FILE=$MNEMONIC_DIR/mnemonic.txt 
+BACKUP_DIR=/var/backups/postgresql
+BLOCKCHAIN_DIR=/mnt/blockchains
+OFAC_DATA_DIR=/var/lamassu/ofac
+ID_PHOTO_CARD_DIR=/opt/lamassu-server/idphotocard
+FRONTCAMERA_DIR=/opt/lamassu-server/frontcamera
+
+mkdir -p $NEW_VOLUME_MOUNTING_POINT$CERT_DIR
+mkdir -p $NEW_VOLUME_MOUNTING_POINT$KEY_DIR
+mkdir -p $NEW_VOLUME_MOUNTING_POINT$MNEMONIC_DIR
+mkdir -p $NEW_VOLUME_MOUNTING_POINT/var/backups
+mkdir -p $NEW_VOLUME_MOUNTING_POINT/mnt
+mkdir -p $NEW_VOLUME_MOUNTING_POINT/var/lamassu
+mkdir -p $NEW_VOLUME_MOUNTING_POINT/opt/lamassu-server
```

Comment:
Probably better to organize this instead of prepending the folder everywhere.

stuff like:

`$NEW_VOLUME_MOUNTING_POINT/lamassu-server` instead of `$NEW_VOLUME_MOUNTING_POINT/opt/lamassu-server`

Reply:
You mean move everything to $NEW_VOLUME_MOUNTING_POINT/lamassu-server, won't that be a little confusing to know what is what?

Reply:
Oh, we can still keep different folders. Something like:

/newvolume/lamassu/idphotocard
/newvolume/lamassu/ofac
/newvolume/backups
/newvolume/mnemonic
/newvolume/ssl/certs
/newvolume/ssl/private